app/views/plans/index.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="row">
+    <div class="col pl-0 py-4">
+        <h1 class="alt-header">WELCOME</h1>
+    </div>
+</div>
+
+<div class="row">
+    <table class="table table-fixed">
+        <colgroup>
+            <col width="364px" />
+            <col />
+            <col width="180px" />
+            <col width="100px" />
+            <col width="130px" />
+        </colgroup>
+
+        <thead>
+            <tr>
+                <th class="pl-0"> Saved Plans </th>
+                <th> Last Revised </th>
+                <th> </th>
+                <th> </th>
+                <th> </th>
+            </tr>
+        </thead>
+
+        <tbody>
+            <% @plans.each do |plan| %>
+                <tr class="plan-entry">
+                    <td class="py-4"> <%= link_to plan[:name], plan %></a> </td>
```

Comment:
any reason we can't to `plan.name` instead of `plan[:name]`?

Reply:
I'll check. I think I tried that there and it told me that plan.name didn't exist.

Reply:
Well, now it works, so I'm going to go with it.